dialog-db/balancing-distribution.md

```diff
@@ -0,0 +1,785 @@
+---
+title: Improving Size Variance
+date: 2025-07-09
+---
+
+#  Improving Size Variance
+
+Dialog DB is designed with [local-first][] principles, aiming to operate without traditional servers by building on content-addressed storage. Like Git, local replicas can cooperate through remote content-addressed stores. To achieve efficient replication in such settings, dialog uses search tree algorithm that is [history independent][] and resilient to [boundary shifts][]. Narrowing [size variance][] is currently under exploration and subject of this article.
```

Comment:
This doesn't really explain why dialog needs to be this way, but perhaps it doesn't need to since we have a README